### ISSUE TITLE
Wflow.jl and update forcing _FillValue

### ISSUE THREAD

**Issue by verseve** [Deltares/hydromt_wflow#60]
When update forcing in hydromt is used the time variable gets a `_FillValue = NaN` as attribute.  This gives problems when reading the file in Wflow.jl when dispatching on the type of the Vector containing time (`Vector{Union{Missing, Dates.DateTime}}`), which should be `Vector{Dates.DateTime}`.

My suggestion is to remove the `_FillValue` from the coordinates of the NetCDF file in hydromt.

This is probably default/standard when using `xarray`, however besides the issue with Wflow.jl it also violates cf standards.

**Comment by hboisgon** [comment]
Thanks @verseve I'll have a look into this. 
It's already on our mind but we should soon really produce cf compliant netdcf files (this is not the only issue).
I now see that with the latest QGIS version, the netcdf files we produce (also staticmaps) do not open properly.

**Comment by visr** [comment]
> I now see that with the latest QGIS version, the netcdf files we produce (also staticmaps) do not open properly.

That's right, the geotransform is not read, so we end up with a default of (0,0) for the top left corner and size 1 pixels.
I think if you do the equivalent of https://github.com/Deltares/Wflow.jl/pull/152 it should be enough.

On the GDAL issue tracker I read

> Since 3.3.2 we have hardened the way to identify dimension axis to be recognized as geospatial ones

**Comment by DirkEilander** [comment]
This is again issue preventing wflow.jl to run reported by Frederiek yesterday. 

I'm not yet clear about when a `_FillValue = NaN` attribute is added to the time dimension, but it seems that we can  prevent it using encoding with `{'_FillValue': None}` https://github.com/pydata/xarray/pull/1659

**Comment by DirkEilander** [comment]
This issue can be fixed in existing inmap files with ncatted (part of NCO), see below. 
NCO can easiliy be installed on linux from conda-forge (not sure how to install on windows).

`ncatted -O -a _FillValue,time,d,, inmaps.nc`

**Comment by visr** [comment]
Reopening this, because although the issue was fixed for time in #75, the same problem is still present for the spatial coordinate variables. E.g. if I generate a wflow model with the latest versions of hydromt, I get this in both the staticmaps and forcing:

```
        double lat(lat) ;
                lat:_FillValue = NaN ;
                lat:standard_name = "latitude" ;
                lat:long_name = "latitude coordinate" ;
                lat:short_name = "lat" ;
                lat:units = "degrees_north" ;
        double lon(lon) ;
                lon:_FillValue = NaN ;
                lon:standard_name = "longitude" ;
                lon:long_name = "longitude coordinate" ;
                lon:short_name = "lon" ;
                lon:units = "degrees_east" ;
```

This does not directly affect Wflow.jl but it does affect downstream processing. In https://github.com/Deltares/Wflow.jl/pull/215 these attributes are removed from Wflow output.

I checked, and it seems that this is the default behavior of xarray as of version 2022.6.0.

```python
import xarray as xr
da = xr.DataArray([1.5,2.5], dims=("x",), coords={"x": [10.5, 11.5]}, name="data")
da.to_netcdf("data.nc")
```

Using `ncdump -h data.nc` then gives:
```
netcdf data {
dimensions:
        x = 2 ;
variables:
        double x(x) ;
                x:_FillValue = NaN ;
        double data(x) ;
                data :_FillValue = NaN ;
}
```

The `_FillValue` for `data` is fine, but for `x` it is not:

Quoting https://cfconventions.org/cf-conventions/cf-conventions.html#attribute-appendix

> `_FillValue`: Allowed for auxiliary coordinate variables but not allowed for coordinate variables.
